server/server.ts

```diff
@@ -147,24 +145,15 @@ app.get('/stats', (req, res) => {
 });
 
 app.get('/youtube', (req, res) => {
-  Youtube.search.list(
-    { part: 'snippet', type: 'video', maxResults: 25, q: req.query.q },
-    (err: any, data: any) => {
-      if (data && data.items) {
-        const response = data.items.map((video: any) => {
-          return {
-            url: 'https://www.youtube.com/watch?v=' + video.id.videoId,
-            name: video.snippet.title,
-            img: video.snippet.thumbnails.default.url,
-          };
-        });
-        res.json(response);
-      } else {
-        console.error(data);
+  if (typeof req.query.q === 'string') {
```

Comment:
is it even possible for this not to be a string after express parses the request?

Reply:
From the typings of Express it could be a string, an array of a string, and some ParsedQs type.

My function expects a string therefore I'm just type checking here.